### PR TITLE
Wire people

### PR DESCRIPTION
**Pull request by tortellinitony**
:cl: Tortellini Tony
add: Adds wire people. So THAT's where all of these cable coils come from! Noose man is an offensive term, actually.
/:cl:

snip snip
https://gfycat.com/MajesticClumsyBactrian

### CODE REVIEW

File: code/game/objects/items/weapons/tools.dm [hunk 1]
```diff
@@ -273,6 +273,24 @@
...
 			user << "<span class='notice'>You are already trying to pull out a teeth!</span>"
 		return
+	if(ishuman(C) && C.dna.species.id == "wire" && C != user)
+		var/mob/living/carbon/human/H = C
+		H.visible_message("<span class='danger'>[user] begins to snip [H] into a bunch of little pieces!</span>", \
+							"<span class='userdanger'>[user] starts to snip you into a bunch of little pieces!</span>")
+		if(!do_mob(user, H, 150))
+			return
+		H.emote("scream")
+		for(var/obj/item/bodypart/L in H.bodyparts)
```
Comment (ghost) [inline code/game/objects/items/weapons/tools.dm]: Please use _istypeless for loops_ you can find more information about this in the [contributing.md](https://github.com/HippieStation/HippieStation13/blob/master/CONTRIBUTING.md)

**Review by ghost — APPROVED** [review]
Looks good but just one minor change ;-)

File: code/modules/hippie/mob/human/species/wire.dm [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/datum/species/wire
+	name = "Noose Man"
+	id = "wire"
+	species_traits = list(NOBREATH,RESISTCOLD,RESISTPRESSURE,NOBLOOD,VIRUSIMMUNE,NOHUNGER,EASYDISMEMBER,MUTCOLORS,EYECOLOR)
+	damage_overlay_type = "synth"
+	mutant_organs = list(/obj/item/organ/tongue/robot)
+	meat = /obj/item/weapon/restraints/handcuffs/cable
+	skinned_type = /obj/item/stack/cable_coil/random
+	teeth_type = /obj/item/stack/cable_coil/cut
+	burnmod = 3.0 //They are super conductive
```
Comment (JohnGinnane) [inline code/modules/hippie/mob/human/species/wire.dm:10]: Should wire people take damage from eletricity?

**Review by JohnGinnane — APPROVED** [review]
👌

File: code/modules/hippie/mob/human/species/wire.dm (hunk 2 above)
Comment (ghost) [reply code/modules/hippie/mob/human/species/wire.dm:10]: Wire people should emit tesla shocks when shocked IMO

**Comment by ghost** [comment]
Roundstart races should be the same across the board gameplay wise.

**Comment by cacogen** [comment]
this is truly bizarre
and not in a good way

File: code/modules/hippie/mob/human/species/wire.dm [hunk 3]
```diff
@@ -0,0 +1,17 @@
+/datum/species/wire
+	name = "Noose Man"
+	id = "wire"
+	species_traits = list(NOBREATH,RESISTCOLD,RESISTPRESSURE,NOBLOOD,VIRUSIMMUNE,NOHUNGER,EASYDISMEMBER,MUTCOLORS,EYECOLOR)
```
Comment (ghost) [inline code/modules/hippie/mob/human/species/wire.dm]: 👎

File: code/game/objects/items/weapons/tools.dm [hunk 4]
```diff
@@ -273,6 +273,24 @@
 		else
 			user << "<span class='notice'>You are already trying to pull out a teeth!</span>"
 		return
+	if(ishuman(C) && C.dna.species.id == "wire" && C != user)
```
Comment (ghost) [inline code/game/objects/items/weapons/tools.dm:277]: :-1:

Comment (tortellinitony) [reply code/game/objects/items/weapons/tools.dm:277]: dude they're made of wire what do you think happens when you cut them up into little pieces with wirecutters?

Comment (ghost) [reply code/game/objects/items/weapons/tools.dm:277]: Snowflake roundstart races should be humans mechanically. Introducing bad game design like this is detrimental.

**Comment by ghost** [comment]
I don't agree with all the special stuff of this race if its selectable roundstart. It can keep all that good stuff if its practically admin-only, such as the plasmamen (?) or otherwise attainable by other means (pod people)

**Comment by KayeArray** [comment]
100 cables, a power cell, defib and some black magic = wireman?

**Comment by Spacedong** [comment]
NOOSE MEN

**Comment by Spacedong** [comment]
unless this is the sprite you can go fuck yourself

ss 2016-01-18 at 07 38 47

**Comment by KayeArray** [comment]
agreed, the current sprites are meh

File: code/game/objects/items/weapons/tools.dm (hunk 4 above)
Comment (cacogen) [reply code/game/objects/items/weapons/tools.dm:277]: if it's a disadvantage, why does it matter?

**Comment by cacogen** [comment]
am i missing a joke about snowflake races or something
why this?

**Comment by tortellinitony** [comment]
https://vgy.me/OE9qxj.png
Sprites from the forum

**Comment by KayeArray** [comment]
it's beautiful

**Comment by Spacedong** [comment]
if it doesn't get the one I posted I gotta -1 it bruv, googly eyes and neon red or go home

**Comment by tortellinitony** [comment]
choose red as your mutant color you autist

**Comment by KayeArray** [comment]
so white colour = sp00kier skeleton?

**Comment by tortellinitony** [comment]
honestly, I don't really wanna see these added

**Pull request closed without merging**